docs/components/TokenLayout.tsx

```diff
@@ -25,7 +25,10 @@ export const TokenLayout = ({
 				sideNav={{
 					title: 'Tokens',
 					titleLink: '/foundations/tokens',
-					items: TOKEN_NAV_LINKS,
+					items: TOKEN_NAV_LINKS.map(({ label, href }) => ({
```

Comment:
Only `href` and `label` need to be passed to `sideNav`. Everything else is invalid.